src/pug/header.pug

```diff
@@ -0,0 +1,117 @@
+                  a.dropdown__linc.plain-text-bold(href='') Контакти
+            span.plain-text-bold.d-none Ми в соціальних мережах:
+            ul.socials
+              li.socials__item
+                a.socials__link(href='')
+                  span.icon-telegram
+              li.socials__item
+                a.socials__link(href='')
+                  span.icon-facebook
+              li.socials__item
+                a.socials__link(href='')
+                  span.icon-instagram
+          div
+            a.pl-50(href='tel: +380991112233') #[img(src='./img/phone.svg' alt='')] (099)11 12 233
+            .dropdown-tel
+              button.dropdown-button.pd-5.toggle-class-js(data-target='.dropdown-tel__content' data-class-to-toggle='dropdown-show')
+                span.icon-vector.pl-5
+              .dropdown-tel__content
+                .left-line
+                ul
+                  li
+                    a.dropdown__linc(href='tel: +380960000000') (096) 00 00 000
+                  li
+                    a.dropdown__linc(href='tel: +380967777777') (095) 77 77 777
+                  li
+                    a.dropdown__linc(href='tel: +380969999999') (096) 99 99 999
+    .row
+      nav.header-nav.col-12
+        ul.plain-text-bold.header-nav__list
+          li.header-nav__item.d-none
+            a.header-nav__link(href='') Головна
+          li.header-nav__item.d-none
+            a.header-nav__link(href='') АкцІї
+          li.header-nav__item.d-none
+            a.header-nav__link(href='') Доставка та оплата
+          li.header-nav__item.d-none
+            a.header-nav__link(href='') Про нас
+          li.header-nav__item.d-none
+            a.header-nav__link(href='') Контакти
+  hr
+  .container.header-cap-lower
+    .row.d-flex
+      .col-6
+        a.logo-linc-min(href='')
+          img.max-width-100(src='./img/logo-min.png' alt='')
+        a.logo-linc(href='')
+          img.max-width-100(src='./img/logo.svg' alt='')
+      .col-6
+        .plain-text-bold.d-flex.justify-end
+          a.shopping-cart(href='')
+            span.icon-shopping-cart.shopping-cart__icon
+            span.shopping-cart__text Кошик
+          .header-button-wrap
+            button.main-button Замовити дзвінок
+    .row
+      .col-12
+        nav.header-nav
+          ul.plain-text-bold.header-nav__list
+            li.header-nav__item.d-none.pl-0
+              a.header-nav__link(href='') Дивани
+            li.header-nav__item.d-none
+              a.header-nav__link(href='') ліжка
+            li.header-nav__item.d-none
+              a.header-nav__link(href='') Крісла
+            li.header-nav__item.d-none
+              a.header-nav__link(href='') комоди
+            li.header-nav__item.d-none
+              a.header-nav__link(href='') шафи
+            li.header-nav__item.d-none
+              a.header-nav__link(href='') Кухня
+  .container.header-cap-lower--xxxl
+    .row.d-flex
+      .col-xl-4.xxxl-col-5
```

Comment:
what does it mean `xxxl-col-5`? did you mean `col-xxxl-5`? check entire document